src/main/java/org/opentestsystem/saaif/item/SaaifItemConstants.java

```diff
@@ -43,20 +43,23 @@
      * The SAAIF item formats.
      */
     interface ItemFormat {
-        String FORMAT_SHORT_ANSWER = "sa";
+        String FORMAT_SA = "sa";
```

Comment:
I'm wondering if you could place the values you have in the enums into a class hierarchy instead (one class for SA, one for WER, etc.).  Then using virtual functions you could provide the correct values for the format, layout, etc.  You could also use virtual function overrides to avoid the switches elsewhere in the code.  Something to consider.